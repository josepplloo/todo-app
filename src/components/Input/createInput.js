import constants from './constants';

function createInput() {
  const inputElement = document.createElement('input');
  inputElement.setAttribute('label', constants.altText);
  inputElement.setAttribute('nameplace', constants.nameplaceText);


  return inputElement;
}

export default createInput;
