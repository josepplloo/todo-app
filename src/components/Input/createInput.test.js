import createInput from './createInput';

it('should return an <input /> element', () => {
  const inputElement = createInput();
  expect(inputElement.tagName).toBe('INPUT');
});

it('should have a styles and layout like the snapshot', () => {
  const inputElement = createInput();
  expect(inputElement).toMatchSnapshot();
});

