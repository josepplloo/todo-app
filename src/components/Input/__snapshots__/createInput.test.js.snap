// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for Input component should be like the snaptshot 1`] = `
<input
  class="app-input"
  label="todo item"
  placeholder=" Write something you need to do"
/>
`;
